src/docs/user/references/GenericInformation.adoc

```diff
@@ -473,6 +473,14 @@ The `submission.mode` Generic Information displays the name of the portal, servi
 
 It can either be one of the standardized values below or a user-defined name.
 
+The user can define a custom value for 'submission.mode' property when submitting the workflow, in studio portal for example.
+
+The values can be seen in Workflow Execution or in Scheduler Portal.
+
+From Workflow Execution the user can see the value in the Generic Information section of the job.
+
+From Scheduler Portal, the value is displayed in "Submitted from" column of the table and in the job info tab.
```

Comment:
I think it is needed to harmonize the naming of this column with what is presented in the health dashboard. Thus, "submission mode" should be used in both documentation and portals